docs/releases.rst

```diff
@@ -4,6 +4,11 @@
     Releases
 ======================
 
+tmt-1.35
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The :ref:`/stories/cli/try/option/epel` ``tmt try`` supports ``--epel`` option
```

Comment:
it looks weird when rendered (the link), maybe some rewording could be useful, maybe with a brief mention that it enables he repo

https://tmt--2985.org.readthedocs.build/en/2985/releases.html#tmt-1-35